### PR TITLE
use StoppableWorkers in the Numato board

### PR DESCRIPTION
**Pull request by penguinland**
I was hoping to get to more boards during 20% Happy Hour today, but this was the only one I finished. 

No changes to behavior are intended: my goal is just to start using StoppableWorkers in all the places that fit this pattern. This one doesn't _quite_ fit perfectly because the background goroutine doesn't actually check a context to see if it should shut down, but it's close enough that I think this is still a useful change (if you disagree, let's talk, and maybe I'll close this PR!).

### CODE REVIEW

File: components/board/numato/board.go [hunk 1]
```diff
@@ -409,8 +410,7 @@ func connect(ctx context.Context, name resource.Name, conf *Config, logger loggi
 
 	b.lines = make(chan string)
 
-	b.activeBackgroundWorkers.Add(1)
-	utils.ManagedGo(b.readThread, b.activeBackgroundWorkers.Done)
+	b.workers = rdkutils.NewStoppableWorkers(b.readThread)
```
Comment (oliviamiller) [inline components/board/numato/board.go:413]: would you be able to have more than one `stoppableworker` in `b.workers` ?

Comment (penguinland) [reply components/board/numato/board.go:413]: Yes, `NewStoppableWorkers` uses a variadic argument:
```go
// Create 3 background goroutines
b.workers = rdkutils.NewStoppableWorkers(b.readThread, b.spinDiscoBall, b.petPuppies)
...
b.workers.Stop() // Block until all 3 goroutines shut down
```

Comment (oliviamiller) [reply components/board/numato/board.go:413]: ah okay cool!

Comment (penguinland) [reply components/board/numato/board.go:413]: (It's possible that to get the genericlinux stuff to work here, I'll also need to add a `b.workers.AddExtraWorker(...)` function. but that doesn't exist yet.)

**Review by oliviamiller — APPROVED** [review]
lgtm!

**Comment by penguinland** [comment]
Rebased to get rid of the bad commit in the history, per Dan's warnings today

**Pull request merged**